dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/AbstractDatasourceProcessor.java

```diff
@@ -86,4 +91,59 @@ public String getDatasourceUniqueId(ConnectionParam connectionParam, DbType dbTy
         BaseConnectionParam baseConnectionParam = (BaseConnectionParam) connectionParam;
         return MessageFormat.format("{0}@{1}@{2}", dbType.getDescp(), baseConnectionParam.getUser(), baseConnectionParam.getJdbcUrl());
     }
+
+    /**
+     * transform other
+     *
+     * @param type type
+     * @param otherMap otherMap
+     */
```

Comment:
This kind of JavaDoc is meaningless, please remove them or write more meaningful JavaDoc

Reply:
Get it and I modify it.